.github/workflows/build.yml

```diff
@@ -29,6 +29,6 @@ jobs:
         with:
           poetry-version: 1.5.1
       - name: "⚙️ Install dependencies"
-        run: "poetry install"
+        run: "poetry install --without dev"
```

Comment:
All changes to github workflows, gitignore, pre-commit-config.yaml, setup.cfg, pyprojekt.toml and so on  should be in a separate PR's. This is to confusing to review. The example python file can stay.

Reply:
Please look at it again. The workflow and formatting changes have been rolled back.